src/utility.js

```diff
@@ -663,6 +663,19 @@ function getItemAttribute(itemData, key) {
   return undefined;
 }
 
+function getReplayTriggersForType(triggers, triggerType) {
+  if (!triggers) return [];
+
+  const filteredTriggers = [];
+  for (const t of triggers) {
+    if (t.type === triggerType) {
+      filteredTriggers.push(t);
+    }
+  }
+
+  return filteredTriggers;
+}
```

Comment:
Does this need to be a function?
```js
const triggers = this.options.recorder?.triggers || [];
const occurrenceTriggers = triggers.filter(t => t.type === 'occurrence');
```
Seems pretty straightforward/self-documenting at the point of usage.

Reply:
It will be called from more that one place and will likely do more work than it does now.